packages/core/content-manager/server/routes/admin.js

```diff
@@ -323,6 +323,38 @@ module.exports = {
         ],
       },
     },
+    {
+      method: 'POST',
+      path: '/collection-types/:model/actions/bulkPublish',
+      handler: 'collection-types.bulkPublish',
+      config: {
+        middlewares: [routing],
+        policies: [
+          'plugin::content-manager.has-draft-and-publish',
+          'admin::isAuthenticatedAdmin',
+          {
+            name: 'plugin::content-manager.hasPermissions',
+            config: { actions: ['plugin::content-manager.explorer.delete'] },
```

Comment:
I think this action should actually be publish instead of delete.
```suggestion
            config: { actions: ['plugin::content-manager.explorer.publish'] },
```
(same for the bulkUnpublish?)